pyproject.toml

```
@@ -40,13 +40,14 @@ slc = [
 ]
 ibis = [ "ibis-framework [exasol] (>=9.1.0, <10)" ]
 transformers = [
-  "exasol-transformers-extension (>=2.2.1, <3)",
+  "exasol-transformers-extension (>=3.0.0, <4)",
   "yaspin (>=3.1.0, <4)",
 ]
-text-ai = [ "exasol-text-ai-extension (>=0.1.0, <2.0.0)" ]
+text-ai = [ "exasol-text-ai-extension (>=0.1.0, <3.0.0)" ]
```

Is this intended, that you increased text-ai to <3.0.0


```suggestion
text-ai = [ "exasol-text-ai-extension (>=0.1.0, <2.0.0)" ]
```

---

that is a little to high, aiming for the stars.. will decrease again